### PR TITLE
Accounts to stage

### LINKED ISSUE #1991: 💡 Homepage optimization
### Start Date

1

### Implementation PR

_No response_

### Reference Issues

_No response_

### Summary

1

### Basic Example

1

### Drawbacks

1

### Unresolved questions

_No response_

### PR DESCRIPTION
**Pull request by darwin808**
# Description

Homepage optimization

Added loading=’lazy’ in image tags 
Remove unused dependencies in package.json
Lazy loaded react components usign next dynamic 
Adding defer to jsdelvr cdn script tag
Uploade assets to zesty dam 

image

### CODE REVIEW

File: src/pages/dashboard.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ export default DashboardPage;
 const cache = {};
 export async function getServerSideProps({ res, resolvedUrl, req }) {
   // this getssrprops should run if login in accounts and docs
-  res.setHeader('Cache-Control', 'private');
+  res.setHeader('Cache-Control', 'private, max-age=6');
```
Comment (darwin808) [inline src/pages/dashboard.js]: revert

File: src/revamp/ui/GetDemoSection/index.js [hunk 2]
```diff
@@ -162,14 +161,62 @@ const GetDemoSection = ({
...
-              <img src={sony} width="91px" height="32px" />
-              <img src={rocketLeague} width="60px" height="32px" />
-              <img src={singlife} width="102.12px" height="32px" />
-              <img src={acorns} width="94px" height="32px" />
-              <img src={phoenixSuns} width="107.54px" height="32px" />
-              <img src={wattpad} width="115.91px" height="32px" />
-              <img src={cornershop} width="96.69px" height="32px" />
-              <img src={bjs} width="36.48px" height="32px" />
+              <img
+                loading="lazy"
```
Comment (darwin808) [inline src/revamp/ui/GetDemoSection/index.js:166]: change alt text to "zesty customer logo COMPANY_NAME"

**Review by darwin808 — COMMENTED** [review]
peer reviewed with @ardeay 

need to change the alt text
revert the brand icon
revert max age

**Comment by ardeay** [comment]
@darwin808 please merge this with those little updates

**Pull request merged**